avro-serializer/src/main/java/io/confluent/kafka/formatter/AvroMessageFormatter.java

```diff
@@ -83,6 +84,8 @@ public void init(Properties props) {
     if (url == null) {
       throw new ConfigException("Missing schema registry url!");
     }
+
```

Comment:
AvroMessageFormatter is actually only used for deserializing messages, so we don't need to modify this class.